hive-metastore/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -156,7 +147,7 @@ protected void doRefresh() {
     String metadataLocation = null;
     try {
       Table table = metaClients.run(client -> client.getTable(database, tableName));
-      validateTableIsIceberg(table, fullName);
+      HiveMetastoreConnector.validateTableIsIceberg(table, fullName);
```

Comment:
`HiveMetastoreConnector` is not need here

Reply:
`validateTableIsIceberg` is a static method and being referred from non-static method.